postgres/src/main/scala/zio/sql/postgresql/PostgresModule.scala

```diff
@@ -128,69 +267,72 @@ trait PostgresModule extends Jdbc { self =>
         case tt @ TUUID           => render(tt.cast(lit.value))                     // todo still broken
         case tt @ TZonedDateTime  => render(tt.cast(lit.value))                     // todo still broken
 
-        case TByte       => render(lit.value)           //default toString is probably ok
-        case TBigDecimal => render(lit.value)           //default toString is probably ok
-        case TBoolean    => render(lit.value)           //default toString is probably ok
-        case TDouble     => render(lit.value)           //default toString is probably ok
-        case TFloat      => render(lit.value)           //default toString is probably ok
-        case TInt        => render(lit.value)           //default toString is probably ok
-        case TLong       => render(lit.value)           //default toString is probably ok
-        case TShort      => render(lit.value)           //default toString is probably ok
-        case TString     => render("'", lit.value, "'") //todo fix escaping
-
-        case _ => render(lit.value) //todo fix add TypeTag.Nullable[_] =>
+        case TByte                => render(lit.value)           //default toString is probably ok
+        case TBigDecimal          => render(lit.value)           //default toString is probably ok
+        case TBoolean             => render(lit.value)           //default toString is probably ok
+        case TDouble              => render(lit.value)           //default toString is probably ok
+        case TFloat               => render(lit.value)           //default toString is probably ok
+        case TInt                 => render(lit.value)           //default toString is probably ok
+        case TLong                => render(lit.value)           //default toString is probably ok
+        case TShort               => render(lit.value)           //default toString is probably ok
+        case TString              => render("'", lit.value, "'") //todo fix escaping
+        case TDialectSpecific(tt) =>
+          tt match {
+            case tt @ TInterval => render(tt.cast(lit.value))
```

Comment:
btw this is where you can render however you like. `TString` is probably a decent example here.